blueprints.yaml

```diff
@@ -39,6 +39,28 @@ form:
       accept:
         - image/*
 
+    use_custom_logo_alt:
```

Comment:
Let's get rid of this setting and just use the `custom_logo_alt` - if there's an alternate logo specified, assume that it should be used.